types/opt/bool_test.go

```diff
@@ -127,3 +129,37 @@ func TestUnmarshalAlloc(t *testing.T) {
 		t.Errorf("got %v allocs, want 0", n)
 	}
 }
+
+func TestBoolFlag(t *testing.T) {
+	tests := []struct {
+		arguments      string
+		wantParseError bool // expect flag.Parse to error
+		want           Bool
+	}{
+		{"", false, Bool("")},
+		{"-test", true, Bool("")},
```

Comment:
What about `--test=""`? Probably the same as this but worth adding maybe

Reply:
done